demosplan/DemosPlanCoreBundle/Resources/config/parameters_default.yml

```diff
@@ -480,4 +480,6 @@ parameters:
     # external_links:
     #     Name of some website: https://link-to-the-website.net
     #     Some other website: https://link-to-some-other-website.net
-    external_links: []
+    external_links:
```

Comment:
The actual values should be defined in the project override, as they are project specific. Here, i'd use the empty array as a default.

Reply:
ok, done here https://github.com/demos-europe/demosplan-core/pull/852/commits/0caa555be3b7f1ab7fefd8d9661048d9ec65cdd9